controllers/netbot.js

```diff
@@ -497,9 +497,9 @@ class netBot extends ControllerBot {
         notifMsg["title-loc-args"] = newArray;
         notifMsg["loc-key"] = alarm.localizedNotificationContentKey();
         notifMsg["loc-args"] = alarm.localizedNotificationContentArray();
-        notifMsg["title_loc_key"] = alarm.localizedNotificationTitleKey();
+        notifMsg["title_loc_key"] = alarm.localizedNotificationTitleKey().replace(/[.:]/g, '_');
```

Comment:
is this absolutely necessary? if changing box side, the localization support for android will be delayed for at least three months.

Reply:
The `-` and `:` can not be used in resource name in android. So we can only fix it in box currently.